frontend/src/app/projects/project-detail/model-overview/model-overview.component.ts

```diff
@@ -4,11 +4,14 @@
  */
 
 import { Component, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
 import { Model, ModelService } from 'src/app/services/model/model.service';
 import {
   Project,
   ProjectService,
 } from 'src/app/services/project/project.service';
+import { SessionService } from '../../../services/session/session.service';
```

Comment:
Prefer absolute (from `src`) paths for imports.

Reply:
:+1: